.github/workflows/main.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches: [ master ]
 env:
   DOKKU_APP_NAME: 'gmmp'
-  DOKKU_HOST: 'dokku-1.hurumap.org'
+  DOKKU_HOST: 'gmmp.dev.codeforafrica.org'
```

Comment:
@kilemensi  do you think we should move this to `secrets` or utilise an action like https://github.com/marketplace/actions/dotenv-action ?

Reply:
We can @esirK but these are not really secret secrets... they're public URLs.